app/Jobs/Ninja/BankTransactionSync.php

```diff
@@ -47,17 +49,35 @@ public function handle()
         foreach (MultiDB::$dbs as $db) {
```

Comment:
@turbo124 do we need any changes here for selfhosted non-multidb envs

Reply:
The way I have managed this so far is to have two loops:

        if (! config('ninja.db.multi_db_enabled')) {
//self hosted
return
}

//hosted multidb loops